tests/test_pipeline_generation.py

```diff
@@ -0,0 +1,43 @@
+'''Test pipeline construction
+    1. Assert lab link to within-schema children
+    2. Assert lab link to subject
+    3. Assert subject link to session
+'''
+
+__all__ = ['pipeline']
+
+from . import pipeline
+
+
+def test_generate_pipeline(pipeline):
+    session = pipeline['session']
+    subject = pipeline['subject']
+    lab = pipeline['lab']
+
+    # test connection Lab->schema children, and Lab->Subject.Lab
+    lab_membership, loc_tbl, subject_lab_tbl = \
+        lab.Lab.children(as_objects=True)
+    assert lab_membership.full_table_name == lab.LabMembership.full_table_name
+    assert loc_tbl.full_table_name == lab.Location.full_table_name
+    assert subject_lab_tbl.full_table_name == \
+        subject.Subject.Lab.full_table_name
+
+    # test connection Subject -> schema children
+    session_tbl, _, subject_line_tbl, subject_protocol_tbl, subject_source_tbl, \
+        subject_strain_tbl, subject_user_tbl, subject_cull_table, subject_death_tbl,\
```

Comment:
```suggestion
        subject_strain_tbl, subject_user_tbl, subject_cull_tbl, subject_death_tbl,\
```

Reply:
Are you suggesting here that we should spell out each instance of `tbl`->`table`? I adopted this format from [wf-ephys](https://github.com/datajoint/workflow-array-ephys/blob/main/tests/test_pipeline_generation.py), but non-words is not PEP8 compliant. I can make the shift systematically

Reply:
Thanks @CBroz1.  No need to make that PEP8 change now.  I was actually just trying to make all variables with the same suffix (`tbl`).